src/metabase/search/scoring.clj

```diff
@@ -14,7 +14,7 @@
 (s/defn normalize :- s/Str
   "Normalize a `query` to lower-case."
   [query :- s/Str]
-  (str/lower-case query))
+  (str/lower-case (str/trim query)))
```

Comment:
I don't know what madman is searching for `   foo   ` instead of `foo`, but…may as well play nice.